less/bin-basket-panel.less

```diff
@@ -0,0 +1,52 @@
+bin-basket.bin-basket-panel {
+  .bin-basket-item {
+    align-items: center;
+    border-bottom: 1px solid rgba(67, 67, 67, 0.17);
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: space-between;
+    padding: 10px 0;
+    .item-title {
+      flex: 0 0 100%;
+      margin: 10px 0;
+    }
+    .item-quantity {
+      width: 50px;
+      input {
+        text-align: center;
+        padding: 0;
+      }
+    }
+    .item-remove {
+      position: unset;
```

Comment:
Be careful, this doesn't always behave well on IE11

Reply:
Good catch!